lib/OpenQA/Git.pm

```diff
@@ -21,16 +21,18 @@ sub config ($self, $args = undef) {
     return $app->config->{'scm git'};
 }
 
-sub _prepare_git_command ($self, $args = undef) {
-    my $dir = $args->{dir} // $self->dir;
+sub _prepare_git_command ($self) {
+    my $dir = $self->dir;
+
     if ($dir !~ /^\//) {
         my $absolute_path = abs_path($dir);
         $dir = $absolute_path if ($absolute_path);
     }
     return ('git', '-C', $dir);
 }
 
-sub _format_git_error ($result, $error_message) {
+sub _format_git_error ($self, $result, $error_message) {
+    my $path = $self->dir;
```

Comment:
I don't see this variable being used. What did you try to achieve here?

Reply:
The path was supposed to go into the error message, I saw it before, but at some point it was removed

Reply:
thanks, this is a leftover

Reply:
for context, we originally wanted to improve the format_git_error method output, however as this message has to match the git output exactly it makes no sense to add or remove any elements to the output

Reply:
What do you mean with "this message has to match the git output exactly"?
Where is the problem?

Reply:
Misunderstanding on my side, sorry. `$dir` is now being used within the output.